README.tfdoc.hcl

```diff
@@ -164,14 +164,14 @@ section {
           END
 
           attribute "allow_all" {
-            type        = bool # TODO: should it be string "TRUE"?
```

Comment:
according to the resource schema these attributes are strings:
```
                        "allow_all": {
                                Type:        schema.TypeString,
                                Optional:    true,
                                Description: "Setting this to true means that all values are allowed. This field can be set only in Policies for list constraints.",
                        },

                        "deny_all": {
                                Type:        schema.TypeString,
                                Optional:    true,
                                Description: "Setting this to true means that all values are denied. This field can be set only in Policies for list constraints.",
                        },

                        "enforce": {
                                Type:        schema.TypeString,
                                Optional:    true,
                                Description: "If `true`, then the `Policy` is enforced. If `false`, then any configuration is acceptable. This field can be set only in Policies for boolean constraints.",
                        },
```

if we have the type as `bool` the module would be opinionated. for simplicity i'd go with `bool` anyway but then please see my comment below.